src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncRewriter.cs

```diff
@@ -173,7 +173,7 @@ protected override BoundStatement GenerateStateMachineCreation(LocalSymbol state
                 bodyBuilder.Add(
                     F.Assignment(
                         F.Field(F.Local(stateMachineVariable), _builderField.AsMember(frameType)),
-                        F.StaticCall(methodScopeAsyncMethodBuilderMemberCollection.BuilderType, "Create", ImmutableArray<TypeSymbol>.Empty)));
+                        F.StaticCall(BinderFlags.IgnoreAccessibility, methodScopeAsyncMethodBuilderMemberCollection.BuilderType, "Create", ImmutableArray<TypeSymbol>.Empty))); // TODO (acasey): conditional
```

Comment:
This was an interim TODO indicating that we shouldn't be hard-coding the flag.  If a TODO includes my alias, it's because I intended to address it before submitting.